project-addons/crm_claim_rma_custom/crm_claim.py

```diff
@@ -34,10 +36,19 @@ class equivalent_products_wizard(models.TransientModel):
 
 
 class CrmClaimRma(models.Model):
-
     _inherit = "crm.claim"
     _order = "id desc"
 
+    @api.multi
+    @api.constrains('category_id')
+    def _check_category_id(self):
+        for claim in self:
+            has_category = False
+            for category in claim.category_id:
+                if category.parent_id.id == 28:
+                    has_category = True
+            claim.bool_category_id = has_category
```

Comment:
Entonces dejo el api.constrains, pero meto este for dentro de una función api.one?